package.json

```diff
@@ -92,6 +92,7 @@
     "100": true,
     "ts": false,
     "jsx": false,
+    "color": false,
```

Comment:
newer node versions colorize the output of errors output to stderr, which breaks tests that are asserting on specific error output